docs/src/main/asciidoc/security-openid-connect-dev-services.adoc

```diff
@@ -51,6 +51,11 @@ include::{includes}/devtools/dev.adoc[]
 ...
 ----
 
+[NOTE]
+====
+Timeout for starting a Keycloak container is 60 secs. You will see an error, `Timed out waiting for log output matching '.*Keycloak.*started.*` if the container failed to start in 60 seconds which may happen due to an image download taking a long time. You can increate the timeout, for example, to 2 minutes: `quarkus.devservices.timeout=2M`.
```

Comment:
It's possible that the Keycloak container does not become ready before the default timeout of 60 seconds. A simple way to overcome the issue is to increase the time out - for example to 2 minutes - using  `quarkus.devservices.timeout=2M``

Reply:
Hi @geoand Thanks, it reads much better, let me update